build-gw-images.sh

```diff
@@ -32,6 +32,9 @@ export TASKCLUSTER_ROOT_URL='https://community-tc.services.mozilla.com'
 unset TASKCLUSTER_CERTIFICATE
 
 export GCP_PROJECT=community-tc-workers
+export AZURE_IMAGE_RESOURCE_GROUP=rg-tc-eng-images
+
+retry az login
```

Comment:
Not _too_ necessary since the `imageset.sh` script will call this if you're not logged in.

Reply:
ahhh i think i did that so it asks at the beginning, not several hours into the task...

Reply:
Perfect reasoning lol. Makes sense thanks